src/ui_fsmenu/addons/rows_ui.cc

```diff
@@ -401,6 +402,7 @@ RemoteAddOnRow::RemoteAddOnRow(Panel* parent,
 		ctrl->install_or_upgrade(info_, false);
 		ctrl->rebuild(true);
 	});
+	install_.set_enabled(info->matches_widelands_version());
```

Comment:
I think developers should still be able to download the add-on, so maybe only disable the Install and Upgrade buttons in release builds